src/sentry/integrations/jira/webhooks.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import absolute_import
+
+from django.views.decorators.csrf import csrf_exempt
+
+from sentry.api.base import Endpoint
+
+from sentry.integrations.atlassian_connect import AtlassianConnectValidationError, get_integration_from_jwt
+from sentry.models import sync_group_assignee_inbound
+
+
+class JiraIssueUpdatedWebhook(Endpoint):
+    authentication_classes = ()
+    permission_classes = ()
+
+    @csrf_exempt
+    def dispatch(self, request, *args, **kwargs):
+        return super(JiraIssueUpdatedWebhook, self).dispatch(request, *args, **kwargs)
+
+    def post(self, request, *args, **kwargs):
+        try:
+            token = request.META['HTTP_AUTHORIZATION'].split(' ')[1]
+        except IndexError:
+            return self.respond(status=400)
+
+        data = request.DATA
+
+        assignee_changed = bool(
```

Comment:
This is what [`any`](https://docs.python.org/2/library/functions.html#any) is for, except you can feed it a lazy generator.

```python
assignee_changed = any(
    item for item in ...
)
```